tools/build.py

```diff
@@ -63,6 +64,7 @@ def devhelp(help):
     parser.add_argument('--static-link', metavar='X', choices=['on', 'off'], default='on', help='enable static linking of jerry command line tool (%(choices)s; default: %(default)s)')
     parser.add_argument('--strip', metavar='X', choices=['on', 'off'], default='on', help='strip release binary (%(choices)s; default: %(default)s)')
     parser.add_argument('--unittests', action='store_const', const='ON', default='OFF', help='build unittests')
+    parser.add_argument('-j', '--jobs', metavar='N', action='store', type=int, default=multiprocessing.cpu_count() + 1, help='Allowed N jobs at once (default: %(default)s)')
```

Comment:
I'd suggest:
* "Allowed N build jobs..."
* moving the option up under '--clean'